config/checkstyle-input-suppressions.xml

```diff
@@ -2483,12 +2483,4 @@
             files="utils[\\/]checkutil[\\/]package-info.java"/>
   <suppress id="UnnecessaryOkComment"
             files="checks[\\/]javadoc[\\/]javadoctype[\\/]InputJavadocType1.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]javadoc[\\/]javadoctype[\\/]InputJavadocType3.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]whitespace[\\/]emptylineseparator[\\/]InputEmptyLineSeparator.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]whitespace[\\/]emptylineseparator[\\/]InputEmptyLineSeparator2.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="utils[\\/]checkutil[\\/]InputCheckUtil7.java"/>
```

Comment:
You may go through previously merged PR for reference. I am attaching a few : 

- https://github.com/checkstyle/checkstyle/pull/14258
- https://github.com/checkstyle/checkstyle/pull/14259
- https://github.com/checkstyle/checkstyle/pull/14260